cmd/cloud/ping.go

```diff
@@ -15,7 +15,11 @@ func newPingCommand() *cobra.Command {
 		Use:   "ping",
 		Short: "Ping the control plane to check authentication and connectivity",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := pingControlPlane()
+			err := getNewAccessTokenUsingRefreshTokenIfCurrentIsExpired()
```

Comment:
I think we are duplicating code and violating DRY principle. Look at Cobra pre-run. You can register some function to be executed before any `vet cloud XXX` command is executed. For that I think you have to use `PersistentPreRun` on the `cmd` for cloud.